### ISSUE TITLE
Accessibility issue with Search field

### ISSUE THREAD

**Issue by zqian** [tl-its-umich-edu/canvas-app-explorer#248]
Brandon Werner reported:

"When typing into the filed, screen readers don't speak that content is dynamically updating, and screen readers may expect, and be confused why they cannot find a search button. 

We recommend setting an aria-live region="polite" on the page containing the number of search results. Aria-live regions are used to tell assistive technology like screen readers about important content changes on a web page. As the user types, the number of search results should be changed in this region which will cause the screen reader to voice them. For maximum compatibility, this text should exist when the page loads. For more information about these regions, see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live."

TEST PLAN:

- Go to CAE tool and enter some text into the Search field
- If there is a match, info text "x out of y tools displayed" will be displayed in the middle. The x is the number of tools matched the search text, and the y is the total count of LTI tools 
- If there is no match, "No matching results" will be displayed instead.

**Comment by roujiawang** [comment]
No unexpected behavior per TEST PLAN instructions.
